### ISSUE TITLE
[Request] auto scroll for text in FEN source select UI

### ISSUE THREAD

**Issue by Nuklear92** [ivarbrandt/skin.fentastic#43]
Hello, is it possible to make the text auto scroll when it's past the limit of the box. For example in skins like Arctic Horizon 1,2, when you're inside the FEN source select UI the text will scroll to show the remaining part that's hidden due to exceeding the limit of the UI box each link has. But when using FENtastic it doesn't and you cannot see the whole name of the link. I know it sounds a bit confusing but if you have AH2 or AH1 to see it for yourself it's pretty easy to recreate, just search any movie and go into the FEN SS UI and you'll see how in AH it auto scrolls the remaining text and in FENtastic it stays static. 

Either way, here are two screenshots one so you can hopefully see more clearly what I mean;

![image](https://github.com/ivarbrandt/skin.fentastic/assets/76958541/fe5e4f8d-8b10-4afb-9280-e0a952b9d8b1)

![image](https://github.com/ivarbrandt/skin.fentastic/assets/76958541/87a26fec-605c-4f13-9794-476dd4d4b427)

Hopefully, this could be something you can implement into the skin so it will auto scroll text, or idk if this is a limitation of the Stuary skin itself and since FENtastic is built from Stuary this is one of the negatives it carries over. 😩

Anyways, this is by no means an obligatory thing to do, I just wanted to share the idea if it's possible to get it working in FENtastic great, if not it's all good either way I can live without it. 👍

**Comment by Nuklear92** [comment]
Oh, that would be awesome! ❤️

**Comment by ivarbrandt** [comment]
Peter coming in hot with the clutch save. 

> But when using FENtastic it doesn't and you cannot see the whole name of the link

I've noticed this too but I didn't really know what to make of it since that window isn't a custom window in the skin.

**Comment by Nuklear92** [comment]
> Peter coming in hot with the clutch save. 
> 
> 
> 
> > But when using FENtastic it doesn't and you cannot see the whole name of the link
> 
> 
> 
> I've noticed this too but I didn't really know what to make of it since that window isn't a custom window in the skin.

Hey, sorry for being out of topic even though this issue would be resolved by Tikipeter. Are these builds you had been releasing with letters at the end "1.0.13k", "1.0.13i", etc. Are these some sort of beta testing builds and not official to the public. I've been wondering since I'm using the official 1.0.13 version and I haven't received any of these builds from the remote repo, lol.🤔😅

**Comment by ivarbrandt** [comment]
> Are these builds you had been releasing with letters at the end "1.0.13k", "1.0.13i", etc. Are these some sort of beta testing builds

Yes, these are test versions that I put only on my testing repo. The repo is primarily for me so I can make sure everything is working correctly before I release the main version to my main Kodi repo. The "official" version that is out right now is the one that you're on, `1.0.13`. Currently, I'm still getting all the bugs squared away for `1.0.14`.

**Comment by Nuklear92** [comment]
Gotcha, thanks for the info! Definitely looking forward to see the improvements.👍
